frontend/src/app/components/footer/Footer.component.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { DigitLayout, DigitDesign } from "@cthit/react-digit-components";
+import styled from "styled-components";
+
+const FooterText = styled.p`
+    font-family: Roboto Thin, sans-serif;
+    font-size: 0.875em;
+    margin-right: auto;
+    margin-left: auto;
+`;
+
+export const Footer = () => (
+    <footer>
+        <DigitLayout.Column justifyContent={"center"}>
```

Comment:
My opinion:
Create a `styled.footer` and make it a flex column. This lessens the amount of elements and depth.